src/deepsparse/evaluation/utils.py

```diff
@@ -79,24 +81,66 @@ def if_generative_language_model(pipeline: Pipeline) -> bool:
     return False
 
 
-def args_to_dict(args: Tuple[Any, ...]) -> Dict[str, Any]:
+def parse_kwarg_tuples(kwargs: tuple) -> Dict:
```

Comment:
This is 1:1 copy of the same function from SparseML. As proposed by @rahul-tuli, in the future let's have it in the `nm-utils` so it can be shared between Deepsparse and SparseML